proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -776,6 +776,16 @@ message DeleteDLQTasksResponse {
     int64 messages_deleted = 1;
 }
 
+message ListQueuesRequest {
+    int32 queue_type = 1;
+    int32 page_size = 2;
+    bytes next_page_token = 3;
+}
+
+message ListQueuesResponse {
+    repeated string queue_names = 1;
```

Comment:
Won't clients need the next page token here?

Reply:
Yes. Added the token.